src/components/step-wrapper/StepWrapper.js

```diff
@@ -23,9 +23,9 @@ const StepWrapper = ({ children, steps, handleSubmit }) => {
     setActiveStep((prev) => prev - 1)
   }
 
-  const stepLabels = steps.map((step, index) => (
+  const stepLabels = steps.map(({ step, isError }, index) => (
```

Comment:
The step has an error, so use the name "hasError."

Reply:
Fixed